types/mssql/index.d.ts

```diff
@@ -242,6 +242,9 @@ export declare class ConnectionError implements Error {
 export interface IColumnOptions {
     nullable?: boolean;
     primary?: boolean;
+    identity?: boolean;
+    readonly?: boolean;
```

Comment:
According to the source on unpkg, `readOnly` is with a capital `O`...

Reply:
good spot, fixed 👍